lib/ifc-utils.ts

```
@@ -1557,81 +1564,137 @@ export function exportData(
+        return `<tr>${cells.join("")}</tr>`;
+      });
+      const table = `<table><thead><tr>${headers
+        .map((h) => `<th>${h}</th>`)
+        .join("")}</tr></thead><tbody>${tableRows.join("")}</tbody></table>`;
+      const html = `<?xml version="1.0" encoding="UTF-8"?>\n<html xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns="http://www.w3.org/TR/REC-html40">\n<head><meta http-equiv="Content-Type" content="application/vnd.ms-excel; charset=UTF-8"/></head><body>${table}</body></html>`;
+      return html;
+    }
+    case "glb": {
+      const scene = new Scene();
+      rows.forEach((element: any) => {
+        if (element.geometry && element.geometry.vertices) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)